src/applications/gqrx/mainwindow.cpp

```diff
@@ -1597,12 +1614,15 @@ void MainWindow::startIqPlayback(const QString& filename, float samprate)
     }
 
     storeSession();
+    backupFreq = ui->freqCtrl->getFrequency();
+    backupOffset = (qint64) rx->get_filter_offset();
 
     auto sri = (int)samprate;
+    auto cf  = (long long) center_freq;
     QString escapedFilename = receiver::escape_filename(filename.toStdString()).c_str();
-    auto devstr = QString("file=%1,rate=%2,throttle=true,repeat=false")
-            .arg(escapedFilename).arg(sri);
-
+    auto devstr = QString("file=%1,rate=%2,freq=%3,throttle=true,repeat=false")
+            .arg(escapedFilename).arg(sri).arg(cf);
+optimize_c
```

Comment:
This line is causing a build failure.

Reply:
I've fixed it.
Editing commits at github without verification is bad thing. I've done a wheel misclick while scrolling up to submit changes.